dll/win32/comctl32/lang/comctl_Hr.rc

```diff
@@ -0,0 +1,86 @@
+/*
+ * PROJECT:     ReactOS MessageBox dialog
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Croatian resource file
+ * TRANSLATORS: Copyright 2022 Andrej Bartulin <andrej.bartulinvr@gmail.com>
+ */
```

Comment:
comctl32 is synced with Wine, you should submit a patch upstream.

Reply:
Ok